.github/workflows/build-wheels.yml

```diff
@@ -0,0 +1,81 @@
+name: Build Wheels
+
+# Only run on new tags starting with `v`
+on: [push]
+  # push:
+  #   tags:
+  #     - 'v*'
+
+jobs:
+  build_wheels:
+    name: Build wheel on ${{ matrix.os }}
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        os: [windows-latest, macos-latest]
+
+    steps:
+    - uses: actions/checkout@v2
+      with:
+        submodules: 'recursive'
+
+    - name: set environment variables
+      uses: allenevans/set-env@v1.1.0
+      with:
+        # Only build on CPython
+        CIBW_BUILD: "cp36-* cp37-* cp38-* cp39-*"
```

Comment:
You can change what Python versions wheels are build for, see https://cibuildwheel.readthedocs.io/en/stable/options/#build-skip